### DISCUSSION TITLE
Use subdomain for mastodon with tag of main domain

### DISCUSSION THREAD

**Discussion by feuerstein-dev** [mastodon/mastodon#18852] [Q&A] [answered]
Hello, i want to host mastodon on my subdomain, but for the tag of the users i would use the main domain without the subdomain:
Example:
**Instead of**
`@user@mastodon.example.com`
**i would rather use:**
`@user@example.com`

Is this possible on mastodon? If yes how?
And how should the server name of the mastodon server be (in the configuration)? `Domain + Subdomain` or only `subdomain`?

And another Question:
Is there a good guide how to install mastodon with docker? I can't find one at the moment.

**Comment by ChlorideCull** [comment, marked as answer]
I may be two years late, but you're looking for [the `WEB_DOMAIN` option.](https://docs.joinmastodon.org/admin/config/#web_domain) Note that it has some extra requirements to do with proxying webfinger.

`LOCAL_DOMAIN` is what shows up in usernames, so `example.com` in your example, while `WEB_DOMAIN` is where mastodon is hosted, so `mastodon.example.com` in your example.

**Reply by feuerstein-dev** [reply]
Thank you, that's exactly the answer I was looking for back then. I don't have a use case for this at the moment, but maybe it will help someone else.

Thank you :)
